### PR TITLE
Adding ID for product description

### PR DESCRIPTION
**Pull request by james-willett**
Adding a HTML ID for the product description, for use in a CSS check in the course

### CODE REVIEW

**Comment by james-willett** [comment]
Hi @notdryft - could you please merge and deploy this small change? thanks!

File: src/main/resources/templates/product_details.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
             class="display-3 mb-5"
             th:text="${product.name} ?: 'Product Name'"
           ></h2>
-          <div class="row" th:utext="${product.description}"></div>
+          <div class="row" id="ProductDescription" th:utext="${product.description}"></div>
```
Comment (Isammoc) [inline src/main/resources/templates/product_details.html:15]: In html, `id`, `class` and other data should be kebab cased.
```suggestion
          <div class="row" id="product-description" th:utext="${product.description}"></div>
```

Comment (james-willett) [reply src/main/resources/templates/product_details.html:15]: @Isammoc is it ok to leave id as `ProductDescription` in this case, else I need to re-record the video?

thanks!

Comment (Isammoc) [reply src/main/resources/templates/product_details.html:15]: It's against best practices. But users will have to manage real applications with downsides too ^^

**Pull request merged**